app/routes/users/components/StudentConfirmation.js

```diff
@@ -101,7 +105,67 @@ const StudentConfirmation = ({
               inputValue="komtek"
             />
           </RadioButtonGroup>
+          <RadioButtonGroup
+            name="isTwoYears"
+            label={
+              <Flex>
+                <div>Begynner du på to-årig master?</div>
+                <div style={{ marginLeft: '5px' }}>
+                  <Tooltip
+                    style={{ marginLeft: '3px' }}
+                    content="Huk av JA her kun hvis du begynner på to-arig master studiet på din respektive linje. Huk av NEI hvis du begynner på 5-årig master."
+                  >
+                    <Icon
+                      name="information-circle-outline"
+                      size={20}
+                      style={{ cursor: 'pointer' }}
+                    />
+                  </Tooltip>
+                </div>
+              </Flex>
+            }
+          >
+            <Field
+              name="isTwoYearsYes"
+              label="Ja"
+              component={RadioButton.Field}
+              inputValue="true"
+            />
+            <Field
+              name="isTwoYearsNo"
+              label="Nei (jeg går 5-årig master)"
+              component={RadioButton.Field}
+              inputValue="false"
+            />
+          </RadioButtonGroup>
           <RadioButtonGroup name="member" label="Vil du bli medlem i Abakus?">
+            <Flex>
+              <div style={{ marginLeft: '5px' }}>
+                <p className={styles.infoText}>
+                  Alle som går Kommunikasjonsteknologi eller Datateknologi kan
+                  bli medlem av Abakus. Du må melde deg inn som medlem for å
+                  kunne delta på arrangementer, kurs og annet Abakus tilbyr. Vi
+                  anbefaler alle nye studenter å melde seg inn. Du kan lese mer
+                  om{' '}
+                  <a
+                    href="https://abakus.no/pages/info-om-abakus"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    Abakus
+                  </a>{' '}
+                  og{' '}
+                  <a
+                    href="https://statutter.abakus.no#medlemskap"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    medlemskapet.
+                  </a>
+                  Det koster ingenting å være medlem i Abakus.
```

Comment:
I think this whole section needs a retouch. With this block of text, it looks a bit like one of them: "Do you want ads" at the bottom of every signup. I think 

- [ ] It should be auto-yes? (_I think so, because that's why you are signing up_)
- [ ] It should actually be at the top, so it's the first thing you read. Then people wont press "Nei"
- [ ] "Du må melde deg inn som medlem for..." should just be "Du må bli medlem for..."

Reply:
Tbh, I agree with the last one, but not the other two. I think that forcing the users to make a decision makes them actually read the text and understand what it means, so people who _should_ select "no" don't just leave it on "yes". And I don't think anyone will press no or yes based on where it is on the page. I think it makes the most sense to have it at the bottom as well.